__snapshots__/linked-versions.js

```diff
@@ -46,6 +46,64 @@ exports['LinkedVersions plugin can skip grouping pull requests 3'] = `
 This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).
 `
 
+exports['LinkedVersions plugin should allow multiple groups of linked versions 1'] = `
+:robot: I have created a release *beep* *boop*
+---
+
+
+<details><summary>pkg1: 1.0.1</summary>
+
+## [1.0.1](https://github.com/fake-owner/fake-repo/compare/pkg1-v1.0.0...pkg1-v1.0.1) (1983-10-10)
+
+
+### Bug Fixes
+
+* some bugfix ([aaaaaa](https://github.com/fake-owner/fake-repo/commit/aaaaaa))
+</details>
+
+<details><summary>pkg4: 1.0.1</summary>
+
+## [1.0.1](https://github.com/fake-owner/fake-repo/compare/pkg4-v1.0.0...pkg4-v1.0.1) (1983-10-10)
+
+
+### Miscellaneous Chores
+
+* **pkg4:** Synchronize second group name versions
+</details>
+
+---
+This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).
+`
+
+exports['LinkedVersions plugin should allow multiple groups of linked versions 2'] = `
+:robot: I have created a release *beep* *boop*
+---
+
+
+<details><summary>pkg2: 0.2.4</summary>
```

Comment:
Am I understanding this feature properly that changes to `pkg3` will also result in changes to `pkg2`, and changes to `pkg1` will also result in updates to `pkg4`?